backend/layers/api/portal_api.py

```diff
@@ -75,12 +75,15 @@ def get_collections_list(from_date: int = None, to_date: int = None, token_info:
     collections = []
     for c in itertools.chain(all_published_collections, all_owned_collections):
         collection = {
-            "id": c.version_id.id if c.published_at is None else c.collection_id.id,
             "visibility": "PRIVATE" if c.published_at is None else "PUBLIC",
             "owner": c.owner,
             "created_at": c.created_at,
         }
-        if c.published_at is None:
+        if c.published_at is not None or c.canonical_collection.originally_published_at is None:
```

Comment:
Not something we need for this PR, but at this point I feel like this condition should be added as a method to `CollectionVersion` (e.g. `.is_new_unpublished()`) since it is ubiquitous

Reply:
I'll put a TODO since adding it will require more unit tests.